packages/apps/purchase-page/pages/promotions-page/thankYou.js

```diff
@@ -82,6 +83,22 @@ class StageThankYou extends React.Component {
       'HtzRusr=; domain=.haaretz.co.il; expires=Thu, 01 Jan 1970 00:00:00 UTC; path=/;';
     document.cookie =
       'TmRusr=; domain=.themarker.com; expires=Thu, 01 Jan 1970 00:00:00 UTC; path=/;';
+
+    if (window && window.sessionStorage.getItem('htz-paypal')) {
```

Comment:
This will throw on the server.

Use 
```suggestion
    if (typeof window !== 'undefined' && window.sessionStorage.getItem('htz-paypal')) {
```